examples/agents/human_in_the_loop/backend/lambda.py

```diff
@@ -0,0 +1,129 @@
+import json
+
+
+def get_named_parameter(event, name):
```

Comment:
i wouldn't normally write code like this as its not defensive if `event` doesn't conform to a schema (like if it's missing the `parameters` attribute), but I see this code is repeated throughout this repo a bunch of times.